mad-dog/src/tests/test_plays.js

```diff
@@ -0,0 +1,92 @@
+const axios = require('axios')
+
+const MaxPollDurationMs = 240000
+
+async function delay (ms) {
+    return new Promise(resolve => setTimeout(resolve, ms))
+}
+
+let numSuccessfullyProcessed = 0
+
+const DEFAULT_INDEX = 0
+
+function randomNumber(min, max) {
+    return Math.floor(Math.random() * (max - min) + min)
+}
+
+async function getTotalPlays() {
+    return (await axios({
+        method: 'get',
+        // use nonce to bypass cache
+        url: `http://localhost:5000/v1/metrics/plays?bucket_size=century&nonce=${randomNumber(1, 10000)}`
+    })).data.data[0].count
+}
+
+async function submitTrackListen(executeOne, trackId, userId) {
+    let success = false
+    let start = Date.now()
+    let totalPlays = await getTotalPlays()
+
+    // Issue write operation
+    try {
+        await executeOne(DEFAULT_INDEX, async (libs) => {
+            await libs.logTrackListen(trackId, userId, null, null, false)
```

Comment:
can we add anonymous play verification to this base case as well as the solana test?